<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="open_pads_file_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>open pads file</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pads_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="excel_generate_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Excel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="excel_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="old_pads_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="open_excel_file_button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Excel File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_old_pads_file_button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Pads File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_pads_information_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>260</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Pads</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
